jobs/debug-pod.Jenkinsfile

```diff
@@ -42,6 +42,9 @@ cosa_img = cosa_img ?: pipeutils.get_cosa_img(pipecfg, params.STREAM)
 
 def stream_info = pipecfg.streams[params.STREAM]
 
+// We'll just always use latest for the controller image here.
+def cosa_controller_img = "quay.io/coreos-assembler/coreos-assembler:main",
```

Comment:
the comma needs to be removed at the end. it is throwing an error I believe

Reply:
fixed